api/capi/include/nnstreamer.h

```diff
@@ -569,6 +574,67 @@ int ml_pipeline_valve_release_handle (ml_pipeline_valve_h valve_handle);
  */
 int ml_pipeline_valve_set_open (ml_pipeline_valve_h valve_handle, bool open);
 
+/********************************************************
+ ** NNStreamer Element Property Control in Pipeline    **
+ ********************************************************/
+
+/**
+ * @brief Gets an element handle in NNStreamer pipelines to control its properties.
+ * @since_tizen 6.0
+ * @remarks If the function succeeds, @a elm_h handle must be released using ml_pipeline_release_element().
```

Comment:
plz check release function. maybe `ml_pipeline_element_release_handle`.

Reply:
I will fix it.